src/elisp/treemacs-workspaces.el

```diff
@@ -138,18 +139,29 @@ Will return `point-min' if there is no next project."
   (inline-quote
    (setq treemacs--project-positions (make-hash-table :test #'equal :size 20))))
 
+(define-inline treemacs-project->key (self)
+  "Get the hash table key of PROJECT."
```

Comment:
PROJECT should be SELF as well.

Reply:
This is now fixed - I also added a comment that `self` does not have to be a project struct.